sda-commons-server-jackson/src/main/java/org/sdase/commons/server/jackson/errors/JerseyValidationExceptionMapper.java

```diff
@@ -43,6 +48,11 @@ public Response toResponse(JerseyViolationException e) {
                           CaseFormat.UPPER_UNDERSCORE,
                           annotation.substring(annotation.lastIndexOf('.') + 1)));
               invalidParameters.add(invalidParameter);
+
+              LOGGER.debug(
```

Comment:
I think if it should help someone, we need info level.

If there are many constraint failures, you get a lot of log messages for a single failed request.
Usually one would introduce toString in the ApiError to log the result properly. But it should not contain all fields.

Reply:
> I think if it should help someone, we need info level.

I changed it to INFO.

Reply:
> If there are many constraint failures, you get a lot of log messages for a single failed request.

Changed it to only one statement.